src/ModalContent/ModalContent.stories.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import Modal from '@mui/core/ModalUnstyled';
+import { ModalContent } from './ModalContent';
+import { Button } from '../Button/Button';
+import { Heading } from '../Heading/Heading';
+
+export default {
+  title: 'Composite/Modal Content',
+  component: ModalContent,
+  subcomponents: {
+    'ModalContent.Body': ModalContent.Body,
+    'ModalContent.Cta': ModalContent.Cta,
+    'ModalContent.Footer': ModalContent.Footer,
+  },
```

Comment:
@Lyafel Please also add `Button` to the list of subcomponents since that's being used in the story.